sources/packages/web/src/types/contracts/DataTableContract.ts

```diff
@@ -180,3 +180,66 @@ export const AvailableToChangeOfferingChangeSummaryHeaders = [
     key: "applicationId",
   },
 ];
+
+/**
+ * In progress application offering change summary header.
+ */
+export const InprogressOfferingChangeSummaryHeaders = [
```

Comment:
As a suggestion, the name of the summary header objects can have `dataTable` in them to be explicit that we are creating them for data table purpose.

Reply:
same here to the file name is `DataTableContract.ts`. So, I thought this better than giving a lengthy name. let me know devs thoughts

Reply:
I missed to notice the file name. I am good here.